src/test/java/stepdefs/SearchStepDefs.java

```diff
@@ -1,54 +1,57 @@
 package stepdefs;
 
+import Pages.GooglePage;
 import io.cucumber.java.After;
 import io.cucumber.java.Before;
 import io.cucumber.java.en.Given;
 import io.cucumber.java.en.Then;
 import io.cucumber.java.en.When;
-import org.openqa.selenium.By;
-import org.openqa.selenium.Keys;
 import org.openqa.selenium.WebDriver;
-import org.openqa.selenium.WebElement;
-import org.openqa.selenium.support.ui.ExpectedConditions;
-import org.openqa.selenium.support.ui.WebDriverWait;
 
-import java.time.Duration;
-import java.util.List;
+public class SearchStepDefs {
 
-import static org.assertj.core.api.Assertions.assertThat;
+    GooglePage google;
+    private WebDriver driver;
 
-public class SearchStepDefs {
-  private WebDriver driver;
-
-  @Before
-  public void setup() {
-    driver = Setup.setupAndGetDriver();
-  }
-
-  @After
-  public void teardown() {
-    driver.quit();
-  }
-
-  @Given("^I am on the Google UK homepage$")
-  public void iAmOnTheGoogleUkHomepage() {
-    driver.manage().window().maximize();
-    driver.get("https://www.google.co.uk");
-  }
-  
-  @When("I enter a search term")
-  public void iEnterASearchTerm() {
-    WebDriverWait wait = new WebDriverWait(driver, Duration.ofSeconds(15));
-    wait.until(ExpectedConditions.elementToBeClickable(By.xpath("//button[@id='L2AGLb2']"))).click();
-    driver.findElement(By.xpath("//input[@title='Search']")).sendKeys("BBC news");
-    driver.findElement(By.xpath("//input[@title='Search']")).sendKeys(Keys.ENTER);
-  }
-
-  @Then("results relevant to the search term are returned")
-  public void resultsRelevantToTheSearchTermAreReturned() {
-    List<WebElement> resultHeaders = driver.findElements(By.xpath("//a/h3"));
-    for(WebElement header : resultHeaders) {
-      assertThat(header.getText()).as("Search results contains search term").contains("BBC");
+    @Before
+    public void setup() {
+        driver = Setup.setupAndGetDriver();
+        google = new GooglePage(driver);
+    }
+
+    @After
+    public void teardown() {
+        driver.quit();
+    }
+
+    @Given("^I am on the Google UK homepage$")
+    public void iAmOnTheGoogleUkHomepage() {
+        google.navigateToGoogle();
+        google.selectRejectAll();
+    }
+
+    @When("^I enter a search term \"BBC news\"$")
```

Comment:
Here, to accept the parameter that we have currently passed through, we need to use the `{string}` within the annotation, see this example:

> @Then("the user should be presented with a {string} gymshark support webpage")
  public void theUserShouldBePresentedWithAGymsharkSupportWebpage(String supportPageTitle) {
    assertThat(PAGE_STATE.getHeaderTextFromWebView()).as("Gymshark Support page has loaded").contains(supportPageTitle);
  }

Reply:
Let me know if that doesn't make sense and we can walk through it 👍

Reply:
@owynowyn made above changes .